Makefile

```diff
@@ -1,5 +1,5 @@
 GOFLAGS :=
-DOCKER_ORG ?= $(USER)
+DOCKER_ORG ?= openshift
```

Comment:
Same as in the other one, please no docker org.

Reply:
done